src/SingleCellAnnotator.imjoy.html

```diff
@@ -847,9 +840,58 @@
     },
     startRedrawAnnotation() {
       this.current_image_data.mask_adjusted = true;
-      this.vector_source.clear()
+      if (this.single_cell_mode){
+        this.vector_source.clear()
+      };
       this.startMaskAnnotation(this.vector_source, this.freehand_mode)
     },
+
+    async restoreAnnotation() {
+      if (!this.selected_single_mask){
+        const features = (new GeoJSON()).readFeatures(this.current_image_data.mask_geojson)
```

Comment:
I doubt this will even work, the variable `features` are scoped inside the if statement, cannot be used outside it. https://stackoverflow.com/questions/40877053/use-if-else-to-declare-a-let-or-const-to-use-after-the-if-else


Here, we should define `let features` outside the if statement since you are going to use it afterwards.